test/inductor/test_halide.py

```diff
@@ -0,0 +1,86 @@
+# Owner(s): ["oncall: pt2"]
+import textwrap
+import unittest
+
+import torch
+from torch._inductor.codecache import HalideCodeCache
+from torch._inductor.runtime.hints import HalideInputSpec, HalideMeta
+from torch._inductor.test_case import run_tests, TestCase
+from torch._inductor.utils import parallel_num_threads
+
+from torch.testing._internal.common_utils import IS_MACOS
+from torch.testing._internal.inductor_utils import HAS_CPU
+
+
+try:
+    import halide
```

Comment:
Should we install halide as a dependency for any of the ci jobs? I dont think this will run at all currently right

Reply:
Yes, do you know any easy way to do this?  There aren't pypi packages for it, so it will need an `apt-get install` or build from source.

Reply:
not sure exactly - maybe `.ci/docker/common/install_base.sh` - there are apt-get calls there